spectrocloud/resource_application_profile.go

```diff
@@ -113,6 +112,12 @@ func resourceApplicationProfile() *schema.Resource {
 								Type: schema.TypeString,
 							},
 						},
+						"install_order": {
+							Type:        schema.TypeInt,
+							Description: "The install orders, will line up installation, If there are multiple manifests placed.",
```

Comment:
@SivaanandM @nikchern is this the install order of App profile layers? Also, is `0` the highest value?

Reply:
Install_order should be an integer value. There are no upper and lower limits. even we can specify negative numbers.

Eg: 
We have a 3-tier application if we specify the install order as 4, -3, and 0 for various tiers. provisioning will happen ascending -3, 0, 4.

Reply:
So based on that explanation, we should update the description so that someone can understand the same thing you explained to me by reading the description.